dumb-jump.el

```diff
@@ -2465,7 +2465,7 @@ Ffrom the ROOT project CONFIG-FILE."
 
 (defun dumb-jump-file-modified-p (path)
```

Comment:
This function is IMO broken, since `modified-file-buffers` is not used and AFAICS this function always returns nil.

Reply:
@phikal looks like this got messed up in #330 around [here](https://github.com/jacktasia/dumb-jump/pull/330/files#diff-7d5fde248d17c1592b9a27209496fe24L2353). Would this be easy for you to fix?

Reply:
Sure. Fixed in #361.